### ISSUE TITLE
Problemi di accessibilità su alcuni componenti

### ISSUE THREAD

**Issue by stefano-unimi** [italia/bootstrap-italia#486]
## Comportamento atteso
Alcuni strumenti di controllo automatico di accessibilità (es siteimprove) segnalano violazioni di linee guida WCAG di livello A e AA

### componente select manca descrizione
Il dropdown che implementa la SELECT non rispetta le regole 1.3.1 Info and Relationships, 3.3.2 Labels or Instructions, 4.1.2 Name, Role, Value
<div class="inner show" role="listbox" id="bs-select-1" tabindex="-1" style=""></div>
Avendo ruolo listbox dovrebbe copiare aria-label o aria-labelledby dalla select originale, ma questo non viene fatto 

#### Possibili soluzioni
Copiare aria-label o aria-labelledby quando si costruisce il dropdown che implementa la select

### componenti button mancano di focus highligh in alcuni casi
I button in un input group non hanno un'evidenza grafica quando hanno il focus, non rispettano la regola 2.4.7 Focus Visible

#### Possibili soluzioni
Modificare gli stili affinché  i controlli che hanno focus siano sempre evidenziati
